test/cpp/end2end/test_service_impl.h

```diff
@@ -451,8 +473,13 @@ class TestMultipleServiceImpl : public RpcService {
     std::unique_lock<std::mutex> lock(mu_);
     return signal_client_;
   }
-  void ClientWaitUntilRpcStarted() { signaller_.ClientWaitUntilRpcStarted(); }
+  int WaitForRpcsWaitingForServerToContinue(
```

Comment:
Please call this something like `ClientWaitUntilNRpcsStarted()`.

Reply:
done